fe/fe-core/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -1379,4 +1379,16 @@
      */
     @ConfField
     public static int grpc_max_message_size_bytes = 1 * 1024 * 1024 * 1024; // 1GB
+
+    @ConfField(mutable = true)
+    public static boolean disable_alter_table = true;
+
+    @ConfField(mutable = true)
+    public static boolean disable_broker_load = false;
+
+    @ConfField(mutable = true)
+    public static boolean disable_truncate_table = false;
+
+    @ConfField(mutable = true)
+    public static boolean disable_stream_load = false;
```

Comment:
I think we can create an array config to do this.

Reply:
Good idea. Array config is easy to be extended and can reduce too much configs.